omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/api/Deletable.java

```diff
@@ -28,4 +28,16 @@
 	 */
 	void delete(String uuid, String reason, RequestContext context) throws ResponseException;
 	
+	/**
+	 * Undeletes the specified resource, which in the OpenMRS context means either unvoiding or
+	 * unretiring it
+	 * 
+	 * @param uuid
+	 * @param context
+	 * @throws ResponseException
+	 * @return unvoided/unretired object
+	 * @since 2.2.0
```

Comment:
2.2.0 is a version of what?

Reply:
Thank you very much for pointing out this @dkayiwa , it was supposed to be 2.25.0 (release version to contain these changes). I have updated the commit to reflect the correction. I initially thought the version to be compared against openmrs-core.